civihr_default_theme/scss/pages/_hr-details.scss

```diff
@@ -65,3 +65,195 @@
     }
   }
 }
+
+.view-mydetails-personal {
+  .contact-id span {
+    @include views-label;
+  }
+
+  .pane-block {
+    border-bottom: 1px solid $gray-light;
+    margin-bottom: 0;
+  }
+
+  .views-row {
+    min-height: 80px;
+    padding-left: 100px;
+    position: relative;
+  }
+
+  .views-field-image-URL {
+    left: 0;
+    position: absolute;
+    top: 0;
+  }
+
+  .views-field-image-URL img {
+    border-radius: 50%;
+    height: 80px;
+    width: 80px;
+  }
+
+  .chr_panel__footer {
+    border-top: 0;
+    margin-top: 0;
+  }
+
+  .views-field-nothing {
+    .field-content {
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: space-between;
+      white-space: nowrap;
+      width: 100%;
+    }
+
+    .contact-id,
+    .gender,
+    .marital-status {
+      margin-right: 1rem;
```

Comment:
Please use pixels for all margin's or padding. Dont use `rem` or `em`. Please fix this in the other places you have added too.

Reply:
Ok, I will change those instances.

Reply:
I still see a lot of `rem` and `em` in the PR diff. Can we remove those?